Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestStepTableViewCell.swift

```diff
@@ -31,20 +34,22 @@ class TestStepTableViewCell: UITableViewCell {
         //create the status image view from the images and set its position and inital display attribute
         var statusIcon: UIImage
         if stepStatus == "PASSED" {
-            statusIcon = UIImage(named: "check_icon_green.png")!
+            iconFileName = "check_icon_green.png"
```

Comment:
This doesn't have anything to do with your code, but maybe we should rename these icons to: check_icon, X_icon, and empty_icon.  The colors at the end were from Jason and me testing icons before Devan created them for us.